### PR TITLE
Re-add get_file_io method

### PR DESCRIPTION
**Pull request by kosiakkatrina**
Accidentally removed it in https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/2689

### CODE REVIEW

**Review by Dinssa — APPROVED** [review]

File: app/services/storage/local_disk_service.rb [hunk 1]
```diff
@@ -13,6 +13,12 @@ def get_file(filename)
       File.open(path, "r")
     end
 
+    def get_file_io(filename)
```
Comment (RachaelBooth) [inline app/services/storage/local_disk_service.rb:16]: Does this mean we've got `get_file` and `get_file_io` that are identical?

Comment (RachaelBooth) [reply app/services/storage/local_disk_service.rb:16]: Ah, I see, they're not quite identical in the s3 version. Cool.

Comment (kosiakkatrina) [reply app/services/storage/local_disk_service.rb:16]: Ah, not anymore...

Comment (kosiakkatrina) [reply app/services/storage/local_disk_service.rb:16]: It needed updating here too!

**Pull request merged**